gymnasium/spaces/dict.py

```diff
@@ -1,18 +1,19 @@
 """Implementation of a space that represents the cartesian product of other spaces as a dictionary."""
+from __future__ import annotations
+
 from collections import OrderedDict
-from collections.abc import Mapping, Sequence
+from collections.abc import Mapping as CollectionsMapping
 from typing import Any
 from typing import Dict as TypingDict
-from typing import List, Optional
-from typing import Sequence as TypingSequence
-from typing import Tuple, Union
+from typing import Mapping as TypingMapping
```

Comment:
I'd say
```python
import collections.abc
import typing
# : collections.abc.Mapping
# : typing.Mapping
```

I know it's already in the code, but `TypingDict` should also be turned into `typing.Dict` etc.

Reply:
Or actually `typing.Dict` isn't necessary anymore now that we have `from __future__ import annotations`

Reply:
Updated